### DISCUSSION TITLE
[Suggestion] Create a Wikipedia page for WezTerm

### DISCUSSION THREAD

**Discussion by mazznoer** [wezterm/wezterm#1040] [General]
And add it to this [list](https://en.m.wikipedia.org/wiki/List_of_terminal_emulators).

**Comment by wez** [comment]
I'm open to it, but I have a conflict of interest in creating the wikipedia page, in that I'm the creator of wezterm.  It might be better if someone else were to go through the process of requesting the page be created.  The notability guidelines seem fairly rigorous as well: https://en.wikipedia.org/wiki/Wikipedia:Notability and require secondary sources.  The Alacritty page has a note that it relies too much on primary sources!

**Comment by ghost** [comment]
Wikipedia has been a bit of a mess for terminals for a good 15-ish years now.  There used to be a large table of many terminals with features noted, it was kind of useful to discover interesting things, but it got continually cut down by editors until what's left looks like a classified ad from a 1997 issue of PC Magazine.

If an editor takes a shine and puts it in themselves, great.  But trying to get stuff in there from outside, even with dead paper cites, is IMHO not worth it.
